packages/framework-provider-azure/src/index.ts

```diff
@@ -13,10 +13,12 @@ import { environmentVarNames } from './constants'
 import { fetchReadModel, storeReadModel } from './library/read-model-adapter'
 import { searchReadModel } from './library/searcher-adapter'
 
+let cosmosClient: CosmosClient
 if (typeof process.env[environmentVarNames.cosmosDbConnectionString] === 'undefined') {
```

Comment:
Why is this a string `'undefined'`?

Reply:
Because it was done fast 😂. But it should just check if the env var exists.